### DISCUSSION TITLE
Can't use recently transferred-in domain - "DNSSEC must be disabled"

### DISCUSSION THREAD

**Discussion by Sam-Scolari** [vercel/community#2407] [Help] [answered]
### Summary

I am trying to use a domain I have recently transferred in to Vercel and my project gives me this error.

"Error: Your domain's nameservers are pointing to Vercel, but you must disable DNSSEC with your domain registrar in order for your nameservers to resolve globally."

I am using whatever default nameservers Vercel uses and I cannot find any option relating to DNSSEC in my domains settings.

![Screenshot_31](https://github.com/vercel/community/assets/56174573/201eef4e-9928-4269-8c2d-84c7f620a0a3)

**Comment by smaeda-ks** [comment]
Hi @Sam-Scolari, I'm taking a look. Will update you shortly

**Reply by smaeda-ks** [reply, marked as answer]
@Sam-Scolari DNSSEC has been disabled. Could you check again?

**Reply by alex-grover** [reply]
hi @smaeda-ks, I'm seeing the same issue after transferring a domain from Google Domains today. is there anything we can do from the end user side to resolve this?

**Reply by smaeda-ks** [reply]
Hi @alex-grover, mind sharing your domain name? I can check

**Reply by alex-grover** [reply]
I just transferred alexgrover.me and have wemovedabroad.com in flight still! thank you 🙏🏽

**Reply by smaeda-ks** [reply]
@alex-grover initiated the process of disabling DNSSEC for `alexgrover.me`. It takes ~24 hours. For the other domain, could you ping me again when the transfer is completed?

**Reply by alex-grover** [reply]
@smaeda-ks it looks like your fix worked but DNSSEC has since been reenabled - alexgrover.me is down once again. Do you mind taking another look?

I canceled the transfer on the other domain and disabled DNSSEC prior to transferring and it seems to be working fine.

**Reply by smaeda-ks** [reply]
hey @alex-grover, sorry to hear that. I'm checking the status with our upstream provider. Will report you back soon

**Reply by smaeda-ks** [reply]
@alex-grover what about now?

**Reply by alex-grover** [reply]
Working now, thank you!

**Reply by quochuynh2412** [reply]
I am having the same issue with letrinhquochuynh.com, I contacted my domain registrar but they said I needed support from Vercel instead of them

**Comment by avault1** [comment]
Hi I transfer my domain kaco.finance from cloudflare to vercel, then get the same error, the registar is GoDaddy.
But I can't find a DNSSEC settings in GoDaddy
How can I resolve this?
